src/graph_export/image_generation_graph_cli_parser.cpp

```diff
@@ -62,10 +62,10 @@ void ImageGenerationGraphCLIParser::createOptions() {
             "Default resolution when not specified by client in a format of WxH; W=width H=height. If not specified, inherited from model.",
             cxxopts::value<std::string>(),
             "DEFAULT_RESOLUTION")
-        ("max_number_images_per_prompt",
+        ("max_num_images_per_prompt",
             "Max allowed number of images client is allowed to request for a given prompt.",
             cxxopts::value<uint32_t>(),
-            "MAX_NUMBER_IMAGES_PER_PROMPT")
+            "MAX_NUM_IMAGES_PER_PROMPT")
         ("default_num_inference_steps",
             "Default number of inference steps when not specified by client.",
```

Comment:
what is the default when not specified?

Reply:
empty. meaning it will not get added to graph.pbtxt. meaning it will be inherited from single point of truth for defaults - image generation options proto, please check this file